aci/resource_aci_dhcprelayp.go

```diff
@@ -81,16 +83,44 @@ func resourceAciDHCPRelayPolicy() *schema.Resource {
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
 						"addr": {
-							Type:         schema.TypeString,
-							Required:     true,
-							ValidateFunc: validation.IsIPv4Address,
+							Type:     schema.TypeString,
+							Required: true,
+							StateFunc: func(val interface{}) string {
+								splitVal := strings.Split(val.(string), "/")
+								if len(splitVal) <= 1 {
+									ip := net.ParseIP(val.(string))
+									return ip.String()
+								} else {
+									ip := net.ParseIP(splitVal[0])
+									return ip.String() + "/" + splitVal[1]
+								}
+							},
+							ValidateFunc: schema.SchemaValidateFunc(validateIPAddress()),
 						},
 					},
 				},
 			},
 		}),
 	}
 }
+
+func validateIPAddress() schema.SchemaValidateFunc {
+	return func(input interface{}, attribute string) (output []string, err []error) {
+		ip_address_input := input.(string)
+		ip := net.ParseIP("0.0.0.0")
```

Comment:
You should be able to initialize it as:
```
var ip []bytes
```

or maybe:
```
var ip net.IP
```

Reply:
Removed this part